javascript-light-bulb/main.js

```diff
@@ -0,0 +1,27 @@
+/*
+  - create storage to access HTML element with bulb class
+  - create storage to access HTML element with container class
+  - create a function for when the bulb is clicked
+    - if bulb class is equal to bulb bulb-dark
+      - set bulb class name to bulb bulb-light
+      - set container class name to container container-light
+    - otherwise
+      - set bulb class name to bulb bulb-dark
+      - set container class name to container container-dark
+  - add an event listener to bulb listening for click event and executing on click function
+*/
+
+var $bulb = document.querySelector('.bulb');
+var $container = document.querySelector('.container');
+
+function onClick(event) {
+  if ($bulb.className === 'bulb bulb-dark') {
```

Comment:
Instead of basing your functionality around the current classes on the DOM element, I would use a variable to keep track of if your lightbulb is "on" or "off". A boolean value would probably be a good choice. Remember what we discussed in Q&A, yesterday, we want all JavaScript functionality to not be reliant on data stored in the DOM as much as possible, you should have the information you need to do your work all centralized in your JavaScript.